src/domain/customer/model/customer.ts

```diff
@@ -1,4 +1,9 @@
-export default interface ICustomer {
+
+abstract class BaseModel {
```

Comment:
Se essa classe BaseModel for comum entre outras classes no futuro, acho que vale tirar ela desse arquivo e pôr em um mais genérico.

Reply:
corrigido